Makefile

```diff
@@ -0,0 +1,25 @@
+BATS := "node_modules/bats/bin/bats"
+
+setup: install-deps launch_registry
+
+install-deps:
+	npm install bats@1.x
+	docker pull busybox
+
+launch_registry:
```

Comment:
also mixing `-` and `_`, any hidden meaning?

Reply:
Trying to be clever. `_` being a continuation and `-` a delimiter. But consistency is better. Changed accordingly.